doc/install/manual-deployment.rst

```diff
@@ -308,25 +308,29 @@ bluestore
 ^^^^^^^^^
 #. Create the OSD. ::
 
-	ssh {node-name}
+	copy /var/lib/ceph/bootstrap-osd/ceph.keyring from monitor node (mon-node1) to /var/lib/ceph/bootstrap-osd/ceph.keyring on osd node (osd-node1)
+	ssh {osd node}
 	sudo ceph-volume lvm create --data {data-path}
 
    For example::
 
-	ssh node1
+    	ssh mon-node1
+    	scp /var/lib/ceph/bootstrap-osd/ceph.keyring root@osd-node1:/var/lib/ceph/bootstrap-osd/ceph.keyring
```

Comment:
Nit: This presupposes that `mon-node1` has ssh credentials for `osd-node1` (and, by extension, other OSD nodes).  I would provide an example using `scp -3` (remote to remote through local "admin" node).